tests/test_main.py

```diff
@@ -143,3 +143,7 @@ def test_table_describe(self):
             {'name': 'alive', 'type': 'bool'},
         ]
         self.assertEqual(columns, expected)
+        
+    def test_sort(self):
```

Comment:
This test, although it passes is not valid as "Edgar Allan Poe" is the first record anyway.